npu.user.js

```diff
@@ -506,8 +507,11 @@ var npu = {
 					var select = $('<ul class="termSelect"></ul>');
 					var stored = npu.getUserData(null, null, ["termSelect", npu.getPage()]);
 					var found = false;
+					var admissionSemester = String($("#lblTrainingName").text().match(/:(\d{4}\/\d{2}\/\d)\[.*?\]\)/)[1]);

```

Comment:
Let's make this a bit more robust by adding an EOL anchor to the end:

    /:(\d{4}\/\d{2}\/\d)\[.*?\]\)$/